packages/client/src/hooks/useTreasury.js

```diff
@@ -220,14 +203,21 @@ const doProposeAddVault = async (treasuryAddr, contractName) => {
       arg(message, t.String),
       arg(keyIds, t.Array(t.UInt64)),
       arg(signatures, t.Array(t.String)),
-      arg(height, t.UInt64)
+      arg(height, t.UInt64),
     ],
     limit: SIGNED_LIMIT,
   });
 };
 
-const doProposeAddCollection = async (treasuryAddr, contractName, contractAddress) => {
-  const intent = `A.${contractAddress.replace("0x", "")}.${contractName}.Collection`;
+const doProposeAddCollection = async (
+  treasuryAddr,
+  contractName,
+  contractAddress
+) => {
+  const intent = `A.${contractAddress.replace(
```

Comment:
nit: add a util function that will do the replacing instead of address.replace("0x","") -> so we make it more readable for the future